src/main/java/org/launchcode/techjobs/persistent/models/GeneratedValue.java

```diff
@@ -0,0 +1,4 @@
+package org.launchcode.techjobs.persistent.models;
+
+public @interface GeneratedValue {
+}
```

Comment:
I think this GeneratedValue interface might have been created by accident while you added the annotation to one of your classes. Deleting this and importing @GeneratedValue from import javax.persistence* will resolve the IllegalArgumentsException you're getting.